src/db.ts

```diff
@@ -281,11 +280,11 @@ export class Db implements OperationParent {
     callback?: Callback<Document>
   ): Promise<Document> | void {
     if (typeof options === 'function') (callback = options), (options = {});
-    options = options || {};
 
+    // Intentionally, we do not inherit options from parent for this operation.
```

Comment:
I think we should add this to the function's docs for the ops that do this. Particularly with BSON options the return value could be unexpectedly different.